modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractScalaCodegen.java

```diff
@@ -113,6 +114,10 @@ public AbstractScalaCodegen() {
     public void processOpts() {
         super.processOpts();
 
+        if (StringUtils.isEmpty(System.getenv("SCALAFMT_PATH"))) {
+            LOGGER.info("Environment variable SCALAFMT_PATH not defined so the Haskell code may not be properly formatted. To define it, try 'export SCALAFMT_PATH=/usr/local/bin/scalafmt' (Linux/Mac)");
```

Comment:
`the Haskell code` do you mean `the Scala code`?

Reply:
Fixed. Sorry for the copy & paste issue.

I did a search and confirmed there's no reference to Haskell.